fdbserver/ResolverInterface.h

```diff
@@ -42,6 +40,8 @@ struct ResolverInterface {
 	RequestStream<struct ResolutionSplitRequest> split;
 
 	RequestStream<ReplyPromise<Void>> waitFailure;
+	// For receiving initial transaction state store broadcast from the master
+	RequestStream<TxnStateRequest> txnState;
```

Comment:
This may not be the right place to ask this question, but some questions about TxnStateStore/TxnStateRequest so I can follow this code:
- IKeyValueStore appears to be storage server related, with plugins to support various types of storage server backends - why "txnStateStore/IKeyValueStore" is a part of "MasterData"? Why does the master needs to know about this?
- Sequencer sends a TxnStateRequest to resolvers (by actor "sendInitialCommitToResolvers()") - this is only in the context of recovery, correct?
- Some context/details about TxnStateRequest would be helpful.
- "EndPoint" appears to be something related to flow. Why is it a part of TxnStateRequest?